pkg/detectors/redis/redis.go

```diff
@@ -18,7 +19,8 @@ type Scanner struct{}
 var _ detectors.Detector = (*Scanner)(nil)
 
 var (
-	keyPat = regexp.MustCompile(`\bredis://[\S]{3,50}:([\S]{3,50})@[-.%\w\/:]+\b`)
+	keyPat = regexp.MustCompile(`\bredi(s{1,2})://[\S]{3,50}:([\S]{3,50})@[-.%\w\/:]+\b`)
```

Comment:
It looks like this added a new capturing group - so I think the match index is probably wrong for the existing detector now. Did you run the test?

Also, please run go fmt

Reply:
formatted and fixed the test, might've been a copy/paste error